docs/en/metadata_oidc_op.rst

```diff
@@ -168,7 +168,7 @@ The EC of an OP MUST configure a metadata of type **"openid_provider"**, that MU
 
 .. warning::
   The OP Metadata of type **"openid_provider"** exposes the claim **jwks** as regulated by OID-FED instead of
-  the claim **jwks_uri** as required at `OpenID.Discovery#OP_Metadata`_.
+  the claim **jwks_uri** as required at `OpenID.Discovery#OP_Metadata`_. However, for interoperability purpose, it MAY expose also the parameter **jwks_uri**.
```

Comment:
noi diciamo CHE DEVE usare jwks ma considerando che una entità può partecipare ad altre federazioni non dobbiamo impedire un modello di interop

possiamo escludere jwks_uri, se necessario, con le metadata policy

Reply:
Stiamo normando il metadata dell'OP in SPID e CIE non in altre federazioni. Inoltre, in quali casi un OP può operare anche in altre federazioni?